tests/unit/all/dependencies-spec.js

```diff
@@ -0,0 +1,47 @@
+/* jshint es3: false, esnext: true */
+/* global jasmine, beforeEach, describe, it, expect, module, inject */
+
+(function () {
+
+    'use strict';
+
+    var moduleNames = [
+        'password',
+        'profile',
+        'registration',
+        'sms-password',
+        'sms-verification',
+        'verification'
+    ];
+
+    moduleNames.forEach(function (moduleName) {
+        describe(moduleName, function () {
+
+            beforeEach(function () {
+
+                module('user_management.' + moduleName);
+
+                this.runModule = function () {
+                    inject(function ($rootScope, $route) {
+                        this.$rootScope = $rootScope;
+                        this.$route = $route;
+                    });
+                };
+
+            });
+
+            it('should load the module without error', function () {
+                expect(function () {
+                    this.runModule();
+                }.bind(this)).not.toThrow();
+            });
+
+            it('should have routes', function () {
+                this.runModule();
+                expect(Object.keys(this.$route.routes)).not.toEqual([]);
```

Comment:
I've thought that before when I first started Angular testing, but it felt like too much like copy-pasting code, and that any route change would require a test change too. I guess as this is a library, the routes shouldn't be changed as often.

Should I test the route definitions, or just that the paths exist? I.e. 
```js
expect($route.routes['/some/route'].someProp).toBe(someValue);
```
or just 
```js
expect($route.routes['/some/route']).toBeDefined();
```